homeassistant/components/launch_library/sensor.py

```diff
@@ -147,7 +163,16 @@ async def async_setup_entry(
             description=description,
             name=name if description.key == "next_launch" else None,
         )
-        for description in SENSOR_DESCRIPTIONS
+        for description in NEXT_LAUNCH_SENSOR_DESCRIPTIONS
+    )
+
+    async_add_entities(
+        StarshipSensor(
+            coordinator=coordinator,
+            entry_id=entry.entry_id,
+            description=description,
+        )
+        for description in STARSHIP_SENSOR_DESCRIPTIONS
     )
```

Comment:
Can this be done in a better way? mypy flags it, as a list can only contain one type. Can I pass a tuple instead?

Reply:
We do not need a separate class for the Starship launch, we can just check if `entity_description.key == "starship_launch"` and set `_next_launch` from  `self.coordinator.data[STARSHIP_EVENTS].upcoming.launches` instead of `self.coordinator.data[UPCOMING_LAUNCHES]` aside for that detail those classes are actually identical

Reply:
You are right, gives a much cleaner look. Thanks!